torchci/pages/metrics.tsx

```diff
@@ -359,12 +359,25 @@ export default function Page() {
               badThreshold={(value) => value > 10}
             />
             <ScalarPanel
-              title={"Pull workflow duration avg"}
-              queryName={"workflow_duration_avg"}
+              title={"Pull workflow duration p50"}
```

Comment:
Since this uses the same query and stuff as the p90, you can probably extract it out to it's own functional component.